### PR TITLE
Add a blog post on Cloud in a box

### PR DESCRIPTION
**Pull request by itrich**
I started to write a blog post about the *Cloud in a box* in kind of a *diary* form. I want to create an entertaining readthrough on our idea of the CiaB with some technical background.

There is still missing many many content, especially on how to use the system. I'll continue to write this guide these days.

@fkr @garloff @berendt If you want to contribute to this posting, please feel free to amend and add yourself to the list of authors!

Sidequestion: Do we have any nice picture of the working session during the first Hackathon?

### CODE REVIEW

**Comment by berendt** [comment]
Release this before R4.

**Comment by itrich** [comment]
As discussed with @garloff, we splitted the blog part into two postings. Maybe we can add now or later a third part that describes how to create a k8s cluster within the CiaB?

We should aim Wednesday, 2023-03-15 for publication.

Please read carefully through the post and feel free to review this PR.

File: _i18n/en/_posts/blog/2023-03-15-ciab-2.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+```
+with `ciab-certificates.crt` in the same directory copied over from the
+CiaB manager host's `/etc/ssl/certs/ca-certificates.crt`.
+
+You can of course also use Terraform and other Infrastructure-as-Code tools
+with these credentials.
+
+## So what to use this for?
+
+You can do what you could what you would do with access to a SCS IaaS system.
```
Comment (jschoone) [inline _i18n/en/_posts/blog/2023-03-15-ciab-2.md:96]: What does that mean?

**Review by jschoone — APPROVED** [review]
LGTM

File: _i18n/en/_posts/blog/2023-03-15-ciab-2.md (hunk 1 above)
Comment (garloff) [reply _i18n/en/_posts/blog/2023-03-15-ciab-2.md]: I meant to express that you have the normal user privileges -- same as you get if you become a pco customer ... except of course that your cloud is a bit smaller ...
I improved the wording.

**Pull request merged**